tests/tmisctests.nim

```diff
@@ -84,11 +84,11 @@ suite "misc tests":
       const warningsToCheck = [
         "[UnusedImport]",
         "[DuplicateModuleImport]",
-        "[Deprecated]",
+        # "[Deprecated]", # todo fixme
```

Comment:
There are some `nimPreviewSlimSystem` deprecation messages which can be fixed later.

Reply:
Right now things seem to outright fail if the user has `nimPreviewSlimSystem` activated in their user config.. https://github.com/nim-lang/nimble/issues/1050